### PR TITLE
created contact table

### PR DESCRIPTION
**Pull request by paulminhdai**
I just created the contact table.

### CODE REVIEW

File: backend/database/contact.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+import os
+
+DB_PASSWORD = os.environ.get('DB_PASSWORD')
```
Comment (lekhuong07) [inline backend/database/contact.py]: From this line

File: backend/database/contact.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+print(DB_PASSWORD)
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://postgres:{pw}@localhost:5432/clubs_api'.format(pw=DB_PASSWORD)
+app.config['SQLALCHEMY_ECHO'] = True
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+
+db = SQLAlchemy(app)
```
Comment (lekhuong07) [inline backend/database/contact.py]: To this line, shouldn't be in this file

File: backend/database/contact.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    def __init__(self, dob, phone, address, state, zipcode, country):
+        self.dob = dob
+        self.phone = phone
+        self.address = address
+        self.state = state
+        self.zipcode = zipcode
+        self.country = country
+
+
+db.create_all()
```
Comment (lekhuong07) [inline backend/database/contact.py]: This line also.

**Review by lekhuong07 — COMMENTED** [review]
Remember to add unitest

File: backend/database/contact.py (hunk 3 above)
Comment (paulminhdai) [reply backend/database/contact.py]: what is unitest are you talking about?

Comment (lekhuong07) [reply backend/database/contact.py]: pull the new version of the repo, you will see it.

**Pull request closed without merging**